docs/diagrams/FindImplementationPredicateCreationSequenceDiagram.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant "f:FindCommand" as FindCommand LOGIC_COLOR
+participant "p:PersonDetailPredicate" as PersonDetailPredicate LOGIC_COLOR
+participant "p:ExamPredicate" as ExamPredicate LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "m:Model" as Model MODEL_COLOR
+end box
+
+mainframe **sd**    create predicate and update the filtered person list <U+0020>
+
+' Create an alt frame to show the predicate creation
+alt prefix is mt| or lt|
+    create ExamPredicate
+    FindCommand -> ExamPredicate : ExamPredicate(...)
```

Comment:
I think u forgot to activate the FindCommand here

Reply:
according to the example given in the textbook on the sd and ref frames, there isnt a need to include the activation bar here